cli/analytics.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/bitrise-io/bitrise/tools/filterwriter"
+	"github.com/bitrise-io/envman/env"
+	"github.com/bitrise-io/envman/models"
+)
+
+func expandStepInputsForAnalytics(environment, inputs []models.EnvironmentItemModel, secrets []string) (map[string]string, error) {
+	sideEffects, err := env.GetDeclarationsSideEffects(environment, &env.DefaultEnvironmentSource{})
+	if err != nil {
+		return map[string]string{}, fmt.Errorf("expandStepInputsForAnalytics() failed, %s", err)
+	}
+
+	// Filter inputs from enviroments
+	expandedInputs := make(map[string]string)
+	for _, input := range inputs {
+		inputKey, _, err := input.GetKeyValuePair()
+		if err != nil {
+			return map[string]string{}, fmt.Errorf("expandStepInputsForAnalytics() failed, %s", err)
+		}
+
+		src := bytes.NewReader([]byte(sideEffects.ResultEnvironment[inputKey]))
```

Comment:
might worth to check if inputKey exists in ResultEnvironment

Reply:
Can an input have the "is_skip" property? If yes, than it amy not be in the result environments, if empty.

Reply:
I see, from envman side the property can be set even for step inputs and it will be used at envman expand.
However, AFAIK we do not have a step using this env property.

Reply:
Anyway if inputKey is no in sideEffects.ResultEnvironment, the below couple line is not needed, you can just add to the expandedInputs hash.